examples/reply/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![no_std]
+
+use gstd::{debug, msg};
+
+#[no_mangle]
+pub unsafe extern "C" fn handle() {
```

Comment:
where this example is tested?

Reply:
added this test to  ```pong``` fixture as @ukint-vs advised